src/reducer.test.js

```diff
@@ -0,0 +1,22 @@
+import reducer from './reducer';
+
+import {
+  setRegions,
+} from './actions';
+
+describe('reducer', () => {
+  describe('setRegions', () => {
+    it('should set regions array', () => {
+      const regions = [
+        { id: 101, name: '서울' },
+      ];
+      const initialState = {
+        regions: [],
+      };
+
+      const state = reducer(initialState, setRegions(regions));
+
+      expect(state.regions).not.toHaveLength(0);
```

Comment:
```suggestion
      expect(state.regions).toHaveLength(1);
```

여긴 명시적으로 길이 1을 해줘도 좋을 것 같네요.